solutions/landing-zone-namespaced/landing-zone/services.yaml

```diff
@@ -0,0 +1,27 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: blueprints.cloud.google.com/v1alpha1
+kind: ProjectServiceSet
+metadata:
+  name: management-project-id # kpt-set: ${management-project-id}
+  namespace: config-control # kpt-set: ${management-namespace}
```

Comment:
For these services would it be best to put them in a special namespace for the host project or keep them as part of a bootstrap script?

Reply:
I vote for the second option,,bootstrap script

Reply:
Sounds good to me.